spec.md

```diff
@@ -1617,6 +1635,48 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 |-----------|-----------|-------------|-------------------|
 | Volume does not exist | 5 NOT_FOUND | Indicates that a volume corresponding to the specified `volume_id` does not exist. | Caller MUST verify that the `volume_id` is correct and that the volume is accessible and has not been deleted before retrying with exponential back off. |
 
+
+#### `ModifyVolume`
+
+A Controller plugin MUST implement this RPC call if plugin has MODIFY_VOLUME controller capability. This RPC allows the CO to change mutable key attributes of a volume. 
+
+SPs SHOULD NOT change attributes from `volume_context` in the `mutable_parameters`.
```

Comment:
This is not the line I had in mind. I'm not sure what the best wording is, because someone would have to work really hard to misunderstand how volume_context works after reading its docs. But any new sentences we add need to go in the CreateVolume docs because that is where volume_context gets created.

Reply:
How about:

```
`volume_context` SHOULD NOT include any fields that is derived from `mutable_parameters`.  `mutable_parameters` and `volume_context` SHOULD be independent from each other.
```

Reply:
I'm torn about whether we need any words at all, because there's no way to express the warning without restating information that's already clearly included in the spec. I'm going to review this one more time assuming no special wording is needed around mutable_parameters and volume_context.

Reply:
Removed the text from CreateVolume for now :)